src/cli.ts

```diff
@@ -52,6 +52,13 @@ export function parseArgs(params: Array<string>): any {
             description:
               "The number of Node.js cluster workers the invoker should use",
             default: 1,
+          })
+          .option("grace", {
+            alias: "g",
+            type: "number",
+            description:
+              "How long to wait for graceful shutdown before exiting forcefully",
+            default: 5000,
```

Comment:
yes, i was going to ask you what a reasonable default might be here.  i chose `5000` since that is the default grace period used by `throng`.  i'll bump it to 15s unless you think something longer (30s/60s) might be more appropriate.

Reply:
Matching `throng` makes sense for a default to me.